app/src/main/java/org/fossasia/openevent/app/data/speakerscall/SpeakersCall.java

```diff
@@ -0,0 +1,59 @@
+package org.fossasia.openevent.app.data.speakerscall;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.github.jasminb.jsonapi.LongIdHandler;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+import com.raizlabs.android.dbflow.annotation.ForeignKey;
+import com.raizlabs.android.dbflow.annotation.ForeignKeyAction;
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+
+import org.fossasia.openevent.app.data.db.configuration.OrgaDatabase;
+import org.fossasia.openevent.app.data.event.Event;
+import org.fossasia.openevent.app.data.event.serializer.ObservableString;
+import org.fossasia.openevent.app.data.event.serializer.ObservableStringDeserializer;
+import org.fossasia.openevent.app.data.event.serializer.ObservableStringSerializer;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
+@Type("speakers-call")
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(exclude = "event")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+@Table(database = OrgaDatabase.class, allFields = true)
+@EqualsAndHashCode()
+@SuppressWarnings("PMD.TooManyFields")
+public class SpeakersCall {
+
+    @Id(LongIdHandler.class)
+    @PrimaryKey
+    public Long id;
+
+    @Relationship("event")
+    @ForeignKey(onDelete = ForeignKeyAction.CASCADE)
+    public Event event;
+
+    public String announcement;
+    public String hash;
+    public String privacy;
+
+    @JsonSerialize(using = ObservableStringSerializer.class)
+    @JsonDeserialize(using = ObservableStringDeserializer.class)
+    public ObservableString startsAt = new ObservableString();
+    @JsonSerialize(using = ObservableStringSerializer.class)
+    @JsonDeserialize(using = ObservableStringDeserializer.class)
+    public ObservableString endsAt = new ObservableString();
```

Comment:
Use standard Date

Reply:
Sorry, I didn't get what do you mean by standard Date ?

Reply:
@iamareebjamal .

Reply:
I meant LocalDate